src/desktop/components/search_bar/view.coffee

```diff
@@ -74,23 +74,20 @@ module.exports = class SearchBarView extends Backbone.View
 
   checkSubmission: (e) ->
     @hideSuggestions()
-
     return if !(e.which is 13) or @selected?
 
-    if e.which is 13 and @enableSpotlightAutocomplete
-      if @spotlightSearchResult
-        location.assign @spotlightSearchResult.href()
-        @$input.val(@spotlightSearchResult.get('display'))
-        @hideSpotlight()
-      else
-        @emptyItemClick()
-      return
-
     unless @isEmpty()
       @trigger 'search:entered', encodeURIComponent(@$input.val())
 
   maybeHideSpotlight: (e) ->
-    return if e.which is 37 or e.which is 39 or e.which is 13 # cursor left, right, enter
+    return unless @enableSpotlightAutocomplete
+    return if e.which is 37 or e.which is 39 # cursor left, right
+
+    if e.which is 13 and @spotlightSearchResult
```

Comment:
This moves our handling of hitting 'Enter' when a spotlight element is selected to `keydown` instead of `keyup`, essentially.

Reply:
would be good to encode these magic numbers as constants

Reply:
👍 what, you dont know the ascii table by heart?